Rubberduck.Parsing/Symbols/DeclarationFinder.cs

```diff
@@ -686,17 +686,23 @@ public Declaration FindMemberEnclosingModule(Declaration callingModule, Declarat
             return null;
         }
 
-        public Declaration FindMemberEnclosingProcedure(Declaration enclosingProcedure, string memberName, DeclarationType memberType, ParserRuleContext onSiteContext = null)
+        public Declaration FindMemberEnclosingProcedure(Declaration enclosingProcedure, string memberName, DeclarationType memberType)
```

Comment:
Since the update changed the line of my previous comment, I repeat it here:
Shouldn't this rather be named `FindMemberInEnclosingProcedure`?

Moreover, I think the parameter was necessary. It is the logic that did not use it that was insufficient.